bsp4s/src/main/scala/ch/epfl/scala/bsp/endpoints/Endpoints.scala

```diff
@@ -3,63 +3,45 @@ package endpoints
 
 import jsonrpc4s.Endpoint
 import jsonrpc4s.Endpoint.unitCodec
-
 object Build extends Build
 trait Build {
-  object initialize
-      extends Endpoint[InitializeBuildParams, InitializeBuildResult]("build/initialize")
-  object initialized extends Endpoint[InitializedBuildParams, Unit]("build/initialized")
-  object exit extends Endpoint[Exit, Unit]("build/exit")
-  object shutdown extends Endpoint[Shutdown, Unit]("build/shutdown")
   object showMessage extends Endpoint[ShowMessageParams, Unit]("build/showMessage")
   object logMessage extends Endpoint[LogMessageParams, Unit]("build/logMessage")
   object publishDiagnostics
       extends Endpoint[PublishDiagnosticsParams, Unit]("build/publishDiagnostics")
   object taskStart extends Endpoint[TaskStartParams, Unit]("build/taskStart")
   object taskProgress extends Endpoint[TaskProgressParams, Unit]("build/taskProgress")
   object taskFinish extends Endpoint[TaskFinishParams, Unit]("build/taskFinish")
+  object initialize
+      extends Endpoint[InitializeBuildParams, InitializeBuildResult]("build/initialize")
+  object initialized extends Endpoint[Unit, Unit]("build/initialized")
+  object shutdown extends Endpoint[Unit, Unit]("build/shutdown")
+  object exit extends Endpoint[Unit, Unit]("build/exit")
 }
-
 object BuildTarget extends BuildTarget
 trait BuildTarget {
-  object compile extends Endpoint[CompileParams, CompileResult]("buildTarget/compile")
-  object test extends Endpoint[TestParams, TestResult]("buildTarget/test")
-  object run extends Endpoint[RunParams, RunResult]("buildTarget/run")
-  object cleanCache extends Endpoint[CleanCacheParams, CleanCacheResult]("buildTarget/cleanCache")
-
   object didChange extends Endpoint[DidChangeBuildTarget, Unit]("buildTarget/didChange")
-
   object sources extends Endpoint[SourcesParams, SourcesResult]("buildTarget/sources")
   object inverseSources
       extends Endpoint[InverseSourcesParams, InverseSourcesResult]("buildTarget/inverseSources")
   object dependencySources
       extends Endpoint[DependencySourcesParams, DependencySourcesResult](
         "buildTarget/dependencySources"
       )
-
   object dependencyModules
       extends Endpoint[DependencyModulesParams, DependencyModulesResult](
         "buildTarget/dependencyModules"
       )
-
   object resources extends Endpoint[ResourcesParams, ResourcesResult]("buildTarget/resources")
-
   object outputPaths
       extends Endpoint[OutputPathsParams, OutputPathsResult]("buildTarget/outputPaths")
-
-  // Scala specific endpoints
-  object scalacOptions
-      extends Endpoint[ScalacOptionsParams, ScalacOptionsResult]("buildTarget/scalacOptions")
-  @deprecated("jvm test environment should be used instead")
-  object scalaTestClasses
-      extends Endpoint[ScalaTestClassesParams, ScalaTestClassesResult](
-        "buildTarget/scalaTestClasses"
-      )
-  @deprecated("jvm run environment should be used instead")
```

Comment:
It looks like we lose these `@deprecated` annotations down below right? Do we want to even have these methods anymore, of if we are breaking things should we just remove them?

Reply:
fixed